### ISSUE TITLE
Cannot read property 'message' of undefined

### ISSUE THREAD

**Issue by huytrongnguyen** [huytrongnguyen/roxie#1]
```
(node:9532) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'message' of undefined
    at Object.exports.logError (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\core\lib\logger.js:6:127)
    at Object.<anonymous> (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:91:52)
    at step (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:43:23)
    at Object.next (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:24:53)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:18:71
    at new Promise (<anonymous>)
    at __awaiter (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:14:12)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:77:83
    at Layer.handle [as handle_request] (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\route.js:137:13)
(node:9532) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9532) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by huytrongnguyen** [comment]
Suggest it should be 
```
export const logError = (msg: string, err?: any) => console.trace(`${nowAsString()} [ERROR] ${msg} caused by:`);
```

**Comment by huytrongnguyen** [comment]
Fixed in `roxie/core` 0.9.5
